---
author: angosht2
author_profile: true
comments: false
date: 2015-10-19 14:55:47+00:00
layout: single
title: Metagenomics and Amplicon Based Sequencing
---

High-throughput sequencing of amplified markers and metagenomics are powerful culture-independent methods that has caused significant changes in the science of microbial ecology. For example, for more than two decades the primary approach to estimate the abundance and taxonomic identity of bacteria/archaea in different samples have been based on amplification and sequencing of the small subunit of ribosomal RNA. The small subunit of ribosomal RNA is universally present in all bacteria/archaea and has a relatively uniform rate of evolution which makes it a good candidate for amplicon based sequencing for classification of a vast diversity of uncultivated microorganisms. However, using the amplicon based sequencing for answering microbial ecology questions has its own shortcomings as well. For example, amplicon sequencing typically only provides insight into the taxonomic composition of the microbial community. It is impossible to directly resolve the biological functions associated with these taxa using this approach. On the other hand, metagenomics, the genomic analysis of a population of microorganisms, provides access to the functional gene composition of the microbial community and therefore gives a much broader description of the microbial community.
Which method would be the best depends on factors like the question we are trying to answer and the costs. Also no matter which method we choose, we need to keep in mind things like sample number, choosing the right "controls", DNA/RNA extraction methods, and the method we choose to analyze the data can affect the outcome significantly.
The following articles give more information on these topics:
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3351745/
http://www.ncbi.nlm.nih.gov/pubmed/23534863
http://www.ncbi.nlm.nih.gov/pubmed/23387867
